src/InterpretAsCommand.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/**
+ * This file is part of graze/telnet-client.
+ *
+ * Copyright (c) 2016 Nature Delivered Ltd. <https://www.graze.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @license https://github.com/graze/telnet-client/blob/master/LICENSE
+ * @link https://github.com/graze/telnet-client
+ */
+
+namespace Graze\TelnetClient;
+
+use Socket\Raw\Socket;
+use Graze\TelnetClient\Exception\UndefinedCommandException;
+
+class InterpretAsCommand
+{
+    /**
+     * @var string
+     */
+    protected $WILL;
+
+    /**
+     * @var string
+     */
+    protected $WONT;
+
+    /**
+     * @var string
+     */
+    protected $DO;
+
+    /**
+     * @var string
+     */
+    protected $DONT;
```

Comment:
Ideally they would be constants, but need to be initialised by a method call so trying to highlight that. Happy to go lc though,